TODO.md

```diff
@@ -1,95 +1,115 @@
 API
+
 - [ ] expose an API callable from C
 
 Async (#126)
 
 Bell
-- [X] bell-style
+
+- [x] bell-style
 
 Color
-- [X] ANSI Colors & Windows 10+
+
+- [x] ANSI Colors & Windows 10+
 - [ ] ANSI Colors & Windows <10 (https://docs.rs/console/0.6.1/console/fn.strip_ansi_codes.html ?
-  https://github.com/mattn/go-colorable/blob/master/colorable_windows.go,
-  https://github.com/mattn/ansicolor-w32.c)
+      https://github.com/mattn/go-colorable/blob/master/colorable_windows.go,
+      https://github.com/mattn/ansicolor-w32.c)
 - [ ] Syntax highlighting (https://github.com/trishume/syntect/)
 - [ ] clicolors spec (https://docs.rs/console/0.6.1/console/fn.colors_enabled.html)
 
 Completion
-- [X] Quoted path
-- [X] Windows escape/unescape space in path
+
+- [x] Quoted path
+- [x] Windows escape/unescape space in path
 - [ ] file completion & escape/unescape (#106)
 - [ ] file completion & tilde (#62)
-- [X] display versus replacement
+- [x] display versus replacement
 - [ ] composite/alternate completer (if the current completer returns nothing, try the next one)
 
 Config
+
 - [ ] Maximum buffer size for the line read
 
 Cursor
+
 - [ ] insert versus overwrite versus command mode
 - [ ] In vi command mode, prevent user from going to end of line. (#94)
 
 Grapheme
+
 - [ ] grapheme & input auto-wrap are buggy
 
 Hints Callback
-- [X] Not implemented on Windows
-- [X] Do an implementation based on previous history
+
+- [x] Not implemented on Windows
+- [x] Do an implementation based on previous history
 
 History
+
 - [ ] Move to the history line n
 - [ ] historyFile: Where to read/write the history at the start and end of
-each line input session.
-- [X] append_history
+      each line input session.
+- [x] append_history
 - [ ] history_truncate_file
 - [ ] custom persistent storage
 
 Input
+
 - [ ] Password input (#58) (https://github.com/conradkdotcom/rpassword) (https://github.com/antirez/linenoise/issues/125)
-- [X] quoted insert (#65)
+- [x] quoted insert (#65)
 - [ ] Overwrite mode (em-toggle-overwrite, vi-replace-mode, rl_insert_mode)
 - [ ] Encoding
-- [X] [Ctrl-][Alt-][Shift-]<Key> (#121)
+- [x] [Ctrl-][alt-][Shift-]<Key> (#121)
```

Comment:
Why `alt` ?

Reply:
Probably a bug with `prettier`. Reverted this.